packages/popup/src/pages/SetPassword/components/SetPasswordForm.js

```diff
@@ -0,0 +1,90 @@
+import {useState} from 'react'
+import {useHistory} from 'react-router-dom'
+import {useTranslation} from 'react-i18next'
+import {PASSWORD_REG_EXP} from '../../../constants'
+import {PasswordInput} from '../../../components'
+import useGlobalStore from '../../../stores'
+import Button from '@cfxjs/component-button'
+
+const validate = value => {
+  return PASSWORD_REG_EXP.test(value)
+}
+
+const SetPasswordForm = () => {
+  const history = useHistory()
+  const {t} = useTranslation()
+  const setCreatedPassword = useGlobalStore(state => state.setCreatedPassword)
+
+  const [password, setPassword] = useState('')
+  const [confirmPassword, setConfirmPassword] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const [confirmErrorMessage, setConfirmErrorMessage] = useState('')
+
+  // TODO: Replace err msg
+  const setInputErrorMessage = value => {
+    if (validate(value)) {
+      setConfirmErrorMessage(
+        value !== confirmPassword ? '输入的密码不一致' : '',
+      )
+      return setErrorMessage('')
+    }
+    setErrorMessage('something wrong')
+  }
+  // TODO: Replace err msg
+  const setConfirmInputErrorMessage = value => {
+    if (password === value) {
+      return setConfirmErrorMessage('')
+    }
+    setConfirmErrorMessage('输入的密码不一致')
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault()
```

Comment:
typo here? why set error message to password here

Reply:
rename the func to fix confusing